document/document.go

```diff
@@ -851,6 +885,32 @@ func (d *Document) onNewRelationship(decMap *zippkg.DecodeMap, target, typ strin
 			rel.TargetAttr = rel.TargetAttr[0:len(rel.TargetAttr)-len(newExt)] + ext
 		}
 
+	//name: zhexiao(肖哲)
+	//date: 2019-10-09
+	//对文档公式数据的解析
+	//================================start
+	case unioffice.OleObjectType:
+		for _, f := range files {
+			if f == nil {
+				continue
+			}
+
+			//fmt.Printf("%#v \n", rel)
+
+			if f.Name == target {
+				path, err := zippkg.ExtractToDiskTmp(f, d.TmpPath)
+				if err != nil {
+					return err
+				}
+
+				d.OleObjectPaths = append(d.OleObjectPaths, OleObjectPath{
+					rid:  rel.IdAttr,
+					path: path,
+				})
+			}
+		}
+		//================================end
```

Comment:
Please remove the start/end comments.  We can see the diff in GitHub or a difftool quite clearly.

Reply:
Fixed.